### PR TITLE
Add pretty-printed benchmark run summary table.

### PR DESCRIPTION
**Pull request by skschneider**

### CODE REVIEW

File: perfkitbenchmarker/benchmark_status.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    string. Multi-line string summarizing benchmark success statuses. Example:
+        --------------------------------------
+        Name          UID            Status
+        --------------------------------------
+        iperf         iperf0         SUCCEEDED
+        iperf         iperf1         FAILED
+        cluster_boot  cluster_boot0  SKIPPED
+        --------------------------------------
+  """
+  assert (len(run_status_tuples),
```
Comment (cmccoy) [inline perfkitbenchmarker/benchmark_status.py]: Nit: len is superfluous here.

File: tests/benchmark_status_test.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  def testCreateSummaryTable(self):
+    result = benchmark_status._CreateSummaryTable(_STATUS_TUPLES)
+    self.assertEqual(result, _STATUS_TABLE)
+
+
+class CreateSummaryTestCase(unittest.TestCase):
+
+  def testCreateSummary(self):
+    p = mock.patch(target=benchmark_status.__name__ + '._CreateSummaryTable',
+                   return_value=_STATUS_TABLE)
```
Comment (cmccoy) [inline tests/benchmark_status_test.py]: Do you need a patch here? Or can _CreateSummaryTable be called with the inputs?

**Comment by cmccoy** [comment]
Thanks. Two small things.

File: perfkitbenchmarker/benchmark_status.py (hunk 1 above)
Comment (skschneider) [inline perfkitbenchmarker/benchmark_status.py]: Removed.

File: tests/benchmark_status_test.py (hunk 2 above)
Comment (skschneider) [inline tests/benchmark_status_test.py]: Removed.

**Pull request merged**